poseidon/build.rs

```diff
@@ -18,7 +18,7 @@ fn main() {
             "256" => 256,
             _ => panic!("Unsupported security level. Supported levels: 80, 128, 256"),
         },
-        Err(_) => 128,
+        Err(_) => 32,
```

Comment:
isn't it to low for production?

Reply:
it is, but see the discussion on Slack - for 128, the contract won't fit into the limits

Reply:
oh, now it won't be needed to store it in a contract - will revert; thanks!

Reply:
reverted